### PR TITLE
docs: added sign up page

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Sign Up page.

image

### CODE REVIEW

File: stylesheets/signup.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.sign-up-container{
+    background: var(--CARD-BG);
+    min-width: max-content;
+    width: fit-content;
+    border-radius: 5px;
+    border: none;
+    margin: 3rem auto;
+    padding: 3rem 3rem 1rem 3rem;
```
Comment (dhimanaishwarya) [inline stylesheets/signup.css:8]: just a suggestion you can use shorthand padding: 3rem 3rem 1rem;

**Review by dhimanaishwarya — APPROVED** [review]
clean code . Looks good to merge

File: stylesheets/signup.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    margin: auto;
+}
+
+.new-user-form input {
+    max-width: 100%;
+    min-width: 40%;
+    width: 20rem;
+    border: thin solid var(--DEFAULT-GREY);
+    border-radius: 4px; 
+    padding: 5px;
```
Comment (Ananyamadhu08) [inline stylesheets/signup.css:32]: use rem instead of px.

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go ⚡️

**Comment by SinchanaMS** [comment]
Thank you for the reviews! Merging to dev.

**Pull request merged**